src/mongo/base/JobBase.class.php

```diff
@@ -96,14 +101,17 @@ public function tearDown()
     /**
      * Resque event when a job failures
      *
-     * @param \Exception  $e   Exception
-     * @param \Resque_Job $job The failed job
+     * @param \Throwable|\Exception $e   Exception or Error
+     * @param \Resque_Job           $job The failed job
      * @return void
      */
-    public static function onFailure(\Exception $e, \Resque_Job $job)
+    public static function onFailure($e, \Resque_Job $job)
```

Comment:
In PHP 7+ this might be called with an `Error` too, which extends `Throwable`. As the library currently supports PHP 5.5 I can't use that keyword, hence removed the typehint.